### PR TITLE
fix duplicated fixtures when load all fixtures environements

### PR DESCRIPTION
**Pull request by KevinArtus**
@alavieille I've add if test to check if functional fixture already exist before to insert production fixtures. Is that is what you expected ?

### CODE REVIEW

**Comment by alavieille** [comment]
why you add a tag empty bc break in comment of your pull request

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php [hunk 1]
```diff
@@ -19,16 +19,18 @@ class LoadWorkflowFunctionDataProduction extends AbstractFixture implements Orde
...
     public function load(ObjectManager $manager)
     {
-        $enName = $this->generateTranslatedValue('en', 'Validator');
-        $frName = $this->generateTranslatedValue('fr', 'Validateur');
-        $workflowFunctionValidator = new WorkflowFunction();
-        $workflowFunctionValidator->addName($enName);
-        $workflowFunctionValidator->addName($frName);
-        $workflowFunctionValidator->addRole($this->getReference('role-production-draft-to-published'));
-        $this->addReference('workflow-function-validator-production', $workflowFunctionValidator);
+        if ($this->hasReference('workflow-function-validator-functional') == false) {
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowFunctionDataProduction.php]: use yoda condition

File: WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowrightDataProduction.php [hunk 2]
```diff
@@ -22,17 +22,19 @@ class LoadWorkflowRightDataProduction extends AbstractFixture implements Ordered
      */
     public function load(ObjectManager $manager)
     {
-        $workflowRight = new WorkflowRight();
-        $workflowRight->setUserId($this->getReference('user-admin')->getId());
+        if ($this->hasReference('workflow-function-validator-production') == true) {
```
Comment (alavieille) [inline WorkflowFunctionModelBundle/DataFixtures/MongoDB/LoadWorkflowrightDataProduction.php]: same for yoda condition

**Comment by KevinArtus** [comment]
@alavieille Bc i do not know wich one i should take :/ I thought this one was a good one.

**Comment by KevinArtus** [comment]
@alavieille yoda condition ok

**Comment by alavieille** [comment]
:+1:  when you have remove the tag OO-BCBREAK of your comment pull request

**Pull request merged**